Quote original thread message if enough time has passed.

nolanlum opened the pull request:
Fixes #54.

nolanlum committed:
Quote original thread message if enough time has passed.

Fixes #54.

Koshroy commented on gateway/message_handler.go:
```diff
@@ -193,11 +158,92 @@ func (sc *SlackClient) handleMessageEvent(incomingChan chan<- *SlackEvent, messa
+		}
+
+		messageText := sc.ParseMessageText(subMessage.Text)
+		for _, attachment := range subMessage.Attachments {
```
If we're overwriting `messageText` each time, why don't we just grab the first or last attachment in `subMessage.Attachments` instead?

Koshroy commented on gateway/message_handler.go:
```diff
@@ -193,11 +158,92 @@ func (sc *SlackClient) handleMessageEvent(incomingChan chan<- *SlackEvent, messa
+		}
+	}
+
+	if messageData.Channel != "" {
```
Do we want to un-nest the conditional here and do what you did above?

```
if messageData.Channel == "" {
    return
}
````

and then do the rest of the logic?

Koshroy reviewed (commented):
Thanks for the refactor!

Copilot committed:
Merge origin/master and address PR review feedback

Co-authored-by: nolanlum <498770+nolanlum@users.noreply.github.com>

Copilot committed:
Fix typo in sent message tracker comment

Agent-Logs-Url: https://github.com/nolanlum/tanya/sessions/3e5e36bc-f9eb-4269-9211-94b8d7691a0c

Co-authored-by: nolanlum <498770+nolanlum@users.noreply.github.com>
